src/clojureEval.ts

```diff
@@ -10,6 +11,17 @@ const BLOCK_DECORATION_TYPE = vscode.window.createTextEditorDecorationType({
     backgroundColor: { id: 'editor.findMatchHighlightBackground' }
 });
 
+const INLINE_RESULT_LENGTH = 150;
```

Comment:
Probably there is a way to get actual visible width of open editor but I didn't find it.

Reply:
Or we could move it to config but I tried to keep config as lean as possible.

Reply:
150 symbols look like a good default to me.